### ISSUE TITLE
Normalize functions hardcoded

### ISSUE THREAD

**Issue by cooperlab** [DigitalSlideArchive/HistomicsDetect#54]
FasterRCNN contains two calls to "tf.keras.applications.resnet.preprocess_input". Normalization method should be a constructor argument since any backbone can be used (not only resnet).
